scripts/us_census/geojsons_low_res/generate_mcf.py

```diff
@@ -63,8 +66,11 @@ def generate_simple_geojsons(self, show=False, eps=None):
                      tempfile.TemporaryFile(mode='r+') as f2:
```

Comment:
Is this line indented correctly?

Reply:
I think so. My linter complains if I move it away from where it is.